config/nginx.conf

```diff
@@ -11,7 +11,8 @@ events {
 
 http {
     # Set lua search path
-    lua_package_path '/code/lua/?.lua;;';
+    lua_package_path '/usr/local/openresty/lualib/?.lua;luarocks/share/lua/5.1/?.lua;luarocks/share/lua/5.1/?/init.lua;resty_modules/lualib/?.lua;lua/?.lua;';
+    lua_package_cpath 'luarocks/lib/lua/5.1/?.so;;';
```

Comment:
extra ";" ?

Reply:
The example I copied from their docs had 2... I don't think it matters though since it seems to work with both 1 or 2